src/actions/transferActions.js

```diff
@@ -50,6 +50,7 @@ function submitDirectTransferTx (txRequest: {
   transferAmount: StandardTokenUnit,
   transferFiatAmountSpot: string,
   fiatType: string,
+  exchangeRate: { cryptoExchangeRate: string, txFeeCryptoExchangeRate: string },
```

Comment:
place exchangeRate definition in .flow file

Reply:
fixed